src/rustls.h

```diff
@@ -1250,6 +1250,15 @@ rustls_result rustls_web_pki_server_cert_verifier_builder_build(struct rustls_we
  */
 void rustls_web_pki_server_cert_verifier_builder_free(struct rustls_web_pki_server_cert_verifier_builder *builder);
 
+/**
+ * Create a new server certificate verifier that uses the system's root store and WebPKI via
+ * `rustls-platform-verifier`.
+ *
+ * The verifier can be used in several `rustls_client_config` instances and must be freed by
+ * the application when no longer needed.
+ */
+rustls_result rustls_platform_server_cert_verifier(struct rustls_server_cert_verifier **verifier_out);
+
 /**
  * Free a `rustls_server_cert_verifier` previously returned from
  * `rustls_server_cert_verifier_builder_build`. Calling with NULL is fine. Must not be
```

Comment:
I think this comment could benefit from an update to mention `rustls_platform_server_cert_verifier` as another source of a verifier to be freed.